extjsdk/src/test/java/io/vantiq/extjsdk/TestUtils.java

```diff
@@ -57,9 +68,53 @@ public void testGetConfigInDir() throws Exception {
             Path p = Files.createFile(Paths.get("serverConfig/server.config"));
             f = new File(p.toString());
             f.deleteOnExit();
-            bw = fillProps(p);
+            bw = fillProps(p, true);
             
-            checkProps();
+            checkProps(false);
+        } finally {
+            if (bw != null) {
+                bw.close();
+            }
+            if (f != null) {
+                //noinspection ResultOfMethodCallIgnored
+                f.delete();
+            }
+            if (dir != null) {
+                //noinspection ResultOfMethodCallIgnored
+                dir.delete();
+            }
+        }
+    }
+
+    @Test
+    public void testGetEnvVar() throws Exception {
+        assumeTrue("\"CONNECTOR_AUTH_TOKEN\" environment variable must be set equal to \"xxxx====\"",
+                envVarAuthToken != null && envVarAuthToken.equals(FAKE_TOKEN));
+        doEnvVarTests(false);
+    }
+
+    @Test
+    public void testGetEnvVarOverwrite() throws Exception {
```

Comment:
How do these tests get run? It appears to be assumed that the user will manually set the various values? It seems like we should create some grade test tasks that set these values as expected so things get tested regardless of the caller's experience.

There are ways in gradle to set environment variables.

Reply:
Sure that makes sense, I'll add it.